inst/include/interface/rcpp/rcpp_objects/rcpp_growth.hpp

```diff
@@ -78,7 +78,7 @@ vectors have been set */
    * @param ages Type vector of ages
    * @return std::map<T, T>
    *
-   * */
+   */
```

Comment:
There are a few `**/` in the codebase (e.g., under FIMS/inst/include/interface/rcpp/rcpp_objects/ folder and fleet.hpp). Not sure if you want to change all of them to `*/`.